plugins/by-name/noice/default.nix

```diff
@@ -1,35 +1,103 @@
 {
   lib,
-  helpers,
-  config,
-  pkgs,
   ...
 }:
-# TODO: This uses a lot of types.anything because noice.nvim types are quite complex.
-# It should be possible to map them to nix, but they would not map really well through
-# toLuaObject, we would maybe need some ad-hoc pre-processing functions.
```

Comment:
Is this comment still relevant? Or was it removed intentionally?

Reply:
I guess I wasn't sure of its relevance anymore if we go down the route of allowing freeform settings and only type the ones that seem obvious. I'm double checking types.

Reply:
The context is that noice types are horrible to map to nix (mixed list/attrs, recursive,...)

With freeform options this is much less of an issue

Reply:
> With freeform options this is much less of an issue

Unless we inadvertently impose type-restrictions by declaring settings options 🙃